### ISSUE TITLE
How do you trigger a reload

### ISSUE THREAD

**Issue by pdavis68** [Mewriick/Blazor.FlexGrid#15]
Now that I have my request parameter, I need to actually use it. I have a search text box I've added above the grid with a button. When the button is clicked, I want trigger the grid to reload and requery my data source passing in the search string. How do I trigger the reload?

**Comment by Mewriick** [comment]
The grid is rerender after you click on button automatically if you have something like this in button 

```
functions {
private string Name;
private LazyLoadingOptions options;

protected override async Task OnInitAsync()
{
    options = new LazyLoadingOptions { DataUri = "/api/SampleData/WeatherForecasts" };
}

void Click()
{
    options.RequestParams.AddParam("Text", "test");
}
}
```

But the problem is now that nobody notify the **TableDaset** to filter data.
This functionality have to be done same way as the **GoToPage** method is implemented
So this mean that I have to internally render the filter button and bind the future filter method from **TableDaset** into button click

Currently I am working on filtering feature and thinking how to solve this problem in nice way

**Comment by pdavis68** [comment]
Adding the parameter makes it re-render (I can see the rendering timings in the console window), but it doesn't re-fetch the data, so nothing changes.

**Comment by Mewriick** [comment]
Yes like I said, I have to implement this like **GoToPage**  method

**Comment by pdavis68** [comment]
Okay. Cool.

**Comment by Mewriick** [comment]
New Filtering feature is available in nuget version 0.7.0 which can cover your scenario

**Comment by aherrick** [comment]
@pdavis68 did you ever get this working?
